backend/app/config.py

```diff
@@ -100,9 +101,9 @@ class Settings(BaseSettings):
     client_origin: str = "http://localhost:3000"
     debug: bool = True
     ow4_base_url: str = "https://dev.online.ntnu.no"
+    ow5_base_url: str = "http://host.docker.internal:4444/api/trpc"
```

Comment:
Maybe we should add a comment above this line that says something like

```suggestion
    # Assuming Vengeful Vineyard is ran inside a Docker container
    ow5_base_url: str = "http://host.docker.internal:4444/api/trpc"
```